server/bookings/deleteBooking.ts

```diff
@@ -46,14 +46,14 @@ export const deleteBooking = async (
   }
 
   const startOfWeek = dateStartOfWeek(booking.date);
+  const lastCancelledTime = canManageOfficeBookings
```

Comment:
Worth getting the tense right here as "cancelledTime" implies it's already been cancelled
```suggestion
  const lastCancelTime = canManageOfficeBookings
```